MdeModulePkg/Core/Dxe/DxeMain/DxeMain.c

```diff
@@ -281,7 +294,15 @@ DxeMain (
   // Start the Image Services.
   //
   Status = CoreInitializeImageServices (HobStart);
-  ASSERT_EFI_ERROR (Status);
+  // MU_CHANGE START Assert Status but omit EFI_NOT_READY as it just implies gCPU is not yet installed
+  if (Status != EFI_NOT_READY) {

```

Comment:
This would assert if Status is EFI_SUCCESS which is not the behavior I think we want.